### PR TITLE
Unify Postprocessing APIs, update PSE Postprocess to follow, change key `shape` to `shape_list`, change arg name `output_polygon` to `box_type` for consistency, support new added models for online inferenece, improve main readme

### PR DESCRIPTION
**Pull request by SamitHuang**
…y name `shape` to `shape_list`, change arg name `output_polygon` to `box_type` for consistency, support new added models for online inferenece, improve main readme (EN)

Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [ ] You have added unit tests

## Motivation

Unify postprocessing protocol, especially for detection, including APIs, naming.  

Postprocessing API protocol is illustrated in [mindocr/postprocess/README.md](https://github.com/SamitHuang/mindocr-1/blob/main/mindocr/postprocess/README.md)

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

**Review by zhtmike — APPROVED** [review]
May I know if inference pipeline supports O2 prediction? As SVTR is optimised for O2 evaluation currently.

**Comment by SamitHuang** [comment]
> May I know if inference pipeline supports O2 prediction? As SVTR is optimised for O2 evaluation currently.

We can add support for O2 for online inference, will add it soon. But O0 should give better results.

**Comment by SamitHuang** [comment]
End2end acc for db++ and svtr is updated to 0.63+ after setting amp_level O2

File: README_CN.md [hunk 1]
```diff
@@ -85,33 +85,40 @@ MindOCR支持多种文本识别模型及数据集，在此我们使用**CRNN**
...
-集成了文本检测、分类和识别串联推理，请参考[此处](docs/cn/inference/inference_tutorial_cn.md)。
+集成了文本检测、分类和识别串联推理。
 
-#### 2.2 使用原生MindSpore推理
+具体说明请参考[MindOCR 310推理](docs/cn/inference/inference_tutorial_cn.md)。
 
-MindOCR支持使用MindOCR训练好的ckpt文件进行文本检测+文本识别串联推理，请参考[此处](toos/infer/text/README.md)。
+#### 2.2 使用原生MindSpore在线推理(CPU/GPU/Ascend 910)
+
+MindOCR提供易用的文本检测识别推理工具，支持CPU/GPU/Ascend 910硬件平台。在线推理基于使用MindOCR训练p完成的模型进行推理。
```
Comment (liangxhao) [inline README_CN.md:97]: 多打了个p ?

Comment (SamitHuang) [reply README_CN.md]: fixed

**Review by liangxhao — APPROVED** [review]
good

**Pull request merged**